docs/core/tools/dotnet-format.md

```diff
@@ -154,3 +154,28 @@ The `dotnet format analyzers` subcommand only runs formatting rules associated w
   ```dotnetcli
   dotnet format --include ./src/ ./tests/ --exclude ./src/submodule-a/
   ```
+* Fix a specific **code style** issue:  
+
+  ```dotnetcli
+  dotnet format style --diagnostics IDExxxx --severity info
```

Comment:
Since this is a section showing examples, it might be better to show an actual ID number rather than IDExxxx